### ISSUE TITLE
Enable GitHub pages

### ISSUE THREAD

**Issue by felixfbecker** [frontarm/async-javascript-cheatsheet#1]
This cheatsheet is great. If GitHub pages were enabled, one could directly view it in the browser in an accessible way (as opposed to the image which doesn't work with a screen reader).

**Comment by jamesknelson** [comment]
I do want to set this up so it's viewable on frontarm.com as HTML instead of an image, but it'll take a little work as the layout is currently hacked together to target "print as pdf" in Chrome.

Regarding accessibility, I think the current markup would need some work too, as the promises are just spans with a CSS class, so they'd be read as "value" instead of "promise to value". That could probably be fixed without my help, and I'd be glad to merge it in.

**Comment by felixfbecker** [comment]
I think it would be great even if it targets Print to PDF currently (would make it easy to do so without having to clone the repo) :)

**Comment by felixfbecker** [comment]
Probably also want to reference the Lato font as a webfont

**Comment by jamesknelson** [comment]
Referencing the Lato font makes sense. Will also need to add Fira Code. Unfortunately its not available on google web fonts so it'll take a little work.

**Comment by devrchancay** [comment]
the dank.sh typography is pretty good too.

**Comment by diomed** [comment]
@jamesknelson 

https://github.com/tonsky/FiraCode/issues/18#issuecomment-382254761
